goodtablesio/app.py

```diff
@@ -1,6 +1,6 @@
 from flask import Flask
-from .blueprints.api import api
-from .blueprints.github import github
+from .plugins.http.blueprint import api
```

Comment:
May be just put `http` plugin to main app or `blueprints.py` file in root? I suppose there is logical difference between core api stuff and integrations/plugins like github.